game-app/game-core/src/main/java/games/strategy/triplea/ui/panels/map/MapPanel.java

```diff
@@ -852,16 +851,15 @@ public void setMouseShadowUnits(final Collection<Unit> units) {
       return;
     }
 
-    final Tuple<BigDecimal, BigDecimal> movementLeft =
+    final Range<BigDecimal> movementLeft =
         getMinAndMaxMovementLeft(
             CollectionUtils.getMatches(units, Matches.unitIsBeingTransported().negate()));
     movementLeftForCurrentUnits =
-        movementLeft.getFirst()
-            + (movementLeft.getSecond().compareTo(movementLeft.getFirst()) > 0 ? "+" : "");
+        movementLeft.getMinimum()
+            + (movementLeft.getMinimum().compareTo(movementLeft.getMaximum()) > 0 ? "+" : "");
```

Comment:
If I'm not mistaken Minimum and Maximum have to be swapped here

Reply:
Good catch, fixed!